File: 2024/day07/day07_test.go
package day07

import (
	"testing"

	"github.com/stretchr/testify/assert"
)

func TestPart1(t *testing.T) {
	assert.Equal(t, 3749, Part1("input_example_1"))
	assert.Equal(t, 20665830408335, Part1("input"))
}

func TestPart2(t *testing.T) {
	assert.Equal(t, 11387, Part2("input_example_1"))
	assert.Equal(t, 354060705047464, Part2("input"))
}

File: 2025/test/day_07_test.exs
defmodule Day07Test do
  use ExUnit.Case

  import TestHelper

  test "part1" do
    assert Day07.part_1(input("day_07_example_1")) == 21
    assert Day07.part_1(input("day_07")) == 1594
  end

  # test "part2" do
  #   assert Day07.part_2(input("day_07_example_1")) == 3_263_827
  #   assert Day07.part_2(input("day_07")) == 12_542_543_681_221
  # end
end
